Core/RoverControllerPresets/DriveControllers/EricSOnController.cs

```diff
@@ -7,6 +7,8 @@ namespace RoverControlApp.Core.RoverControllerPresets.DriveControllers;
 public class EricSOnController : IRoverDriveController
 {
 	private const float TURN_ANGLE = 89;
+	private MqttClasses.KinematicMode Mode = MqttClasses.KinematicMode.Compatibility;
```

Comment:
It's private and not a property so its _mode

Reply:
same as above